### DISCUSSION TITLE
Footnotes?

### DISCUSSION THREAD

**Discussion by jaschaio** [facebook/lexical#5432] [Q&A]
Has anybody been able to add footnotes to lexical? I was looking for it but found absolutely nothing.

**Comment by Gusis93** [comment]
Did you find anything about footnotes? Also trying to add footnotes but can't find anything

**Comment by CanRau** [comment]
Also looking soonish into footnotes, here's potentially some inspiration

- https://github.com/buttondown/tiptap-footnotes
- https://prosemirror.net/examples/footnote/

Think I like the prosemirror example, which is more inline a little better I think 🤔

**Comment by etrepum** [comment]
Conceptually footnotes and comment nodes are very similar, it should be pretty straightforward to implement either inline or block footnotes with the combination of a plugin and a custom DecoratorNode for the cues.

**Comment by 2wheeh** [comment]
Hi guys, I just published https://github.com/2wheeh/lexical-footnote
